src/server/dfly_bench.cc

```diff
@@ -724,6 +802,82 @@ void WatchFiber(atomic_bool* finish_signal, ProactorPool* pp) {
   }
 }
 
+ClusterSpec FetchCluster(const tcp::endpoint& ep, ProactorBase* proactor) {
```

Comment:
I went over this function briefly but I *did not* do a thorough review so I can't really guarantee the correctness of it. Moreover, for changes like this it's nice to have tests. And speaking of that do we have tests for `dfly_bench` ?

Reply:
we do not - it's not part of a production stack